infra/label_harmonizer_step_functions/lambdas/harmonize_labels.py

```
@@ -0,0 +1,566 @@
+        return cache_path
+
+    logger.info(f"Downloading ChromaDB snapshot from s3://{bucket}/{collection}/")
+
+    # Get latest pointer
+    pointer_key = f"{collection}/snapshot/latest-pointer.txt"
+    try:
+        response = s3.get_object(Bucket=bucket, Key=pointer_key)
+        timestamp = response["Body"].read().decode().strip()
+        logger.info(f"Latest snapshot timestamp: {timestamp}")
+    except Exception as e:
+        logger.error(f"Failed to get pointer: {e}")
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)